### ISSUE TITLE
[BUG] Thirst bar is not always displaying

### ISSUE THREAD

**Issue by Akiranya** [PluginBugs/Issues-ItemsAdder#1157]
**Describe the bug**
Thirst bar is not always displaying even if I have activated it but when I drink a bottle, the thirst bar will appear **perfectly** but only last for about 2 sec and it disappears. I tested it in survival mode. The thirst hud option is enabled and I tried /iareload and restart server and it did not work.

**To Reproduce**
The thirst bar just does not show ... not sure how to reproduce it sorry. I'm happy to provide any addtional information.

**Expected behavior**
The thirst bar should be always visiable.

**Screenshots**

**Info (please complete the following information):**
 - Server version: Purpur 1.17.1
 - Client version: 1.17.1

**Special Info**
Please do not write "latest"! Tell us the specific plugin version you are using.
 - ItemsAdder version: Version: 2.4.12
 - LightAPI Fork version: 3.5.2
 - LoneLibs version: 1.0.14
 - ProtocolLib version: 4.7.0

**Additional context**
I've merged a resourcepack with the one of ItemsAdder but the merge seems all good because the texture itself is all good so far. The only issue I have right now is thirst hud not always displaying.

**Comment by Akiranya** [comment]
Closed as I figured it out on Discord. Many thanks to sircraftypants!

**Comment by LoneDev6** [comment]
> 
> 
> Closed as I figured it out on Discord. Many thanks to sircraftypants!

Good!
Feel free to share the solution here, thanks

**Comment by Akiranya** [comment]
Here is the solution :D

TLDR if the server admin wants to use another plugin (or something simialr) to distribute resource pack to players and they don't want to use IA to do the job, then they still have to make sure that one of the `enabled: false` (except the one under `no-host:`) under `hosting:` has to be `enabled: true` AND set `apply-on-join: false`.

```
resource-pack:
  hosting:
    no-host:
      enabled: false
    auto-external-host:
      enabled: false
    self-host:
      enabled: false
      server-ip: 127.0.0.1
      pack-port: 8163
    external-host:
      enabled: true   <---- better set this to be 'true'
      url: https://xxx.zip  <---- if choose another plugin to distribute the resource pack, the url here doesn't really matter even tho there are errors on the console
  apply-on-join: false <---- this has to be 'false' as we want other plugin to distribute the resource pack
```

This way IA won't distribute the resource pack but IA needs it to make the "always visiable hud" work.

**Comment by LoneDev6** [comment]
This is indeed a bug then.
Thanks for explaining that!
In the next update you will be able to set no-host and the hud will still be shown.

```yml
    no-host:
      enabled: true
```
